From 6a1de5dca4e1118ccbb0afb0a561ffc2227f1d5e Mon Sep 17 00:00:00 2001
From: Vadim Rutkovsky <vrutkovs@redhat.com>
Date: Wed, 25 Mar 2015 15:26:18 +0100
Subject: [PATCH] Fix builddir!=srcdir issue

---
 src/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index 49fd76e..14cf9f4 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -42,7 +42,7 @@ gnome_nibbles_SOURCES = \
 BUILT_SOURCES = nibbles-resources.c
 
 nibbles-resources.c: $(shell $(GLIB_COMPILE_RESOURCES) --generate-dependencies nibbles.gresource.xml)
-	$(AM_V_GEN) $(GLIB_COMPILE_RESOURCES) --target=$@ --sourcedir=$(srcdir) --generate-source nibbles.gresource.xml
+	$(AM_V_GEN) $(GLIB_COMPILE_RESOURCES) --target=$@ --sourcedir=$(builddir) --sourcedir=$(srcdir) --generate-source $(srcdir)/nibbles.gresource.xml
 
 gnome_nibbles_CPPFLAGS = \
 	-I$(top_srcdir) \
-- 
1.8.3.1

