From fc6ffc1143dbee30a17d0c31835a44469f13e9ff Mon Sep 17 00:00:00 2001
From: Vadim Rutkovsky <vrutkovs@redhat.com>
Date: Wed, 25 Mar 2015 15:38:46 +0100
Subject: [PATCH] Fix builddir!=srcdir issue

---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index 6f2c039..d5ad737 100644
--- a/configure.ac
+++ b/configure.ac
@@ -108,6 +108,8 @@ AC_CONFIG_FILES([Makefile
 		po/Makefile.in
 		])
 
+echo \#buildapi-variable-no-builddir >/dev/null
+
 dnl libgd
 LIBGD_INIT([
   gtk-hacks
-- 
2.7.4

