From be1090246fbf37da4b257a9bb35312427a6166d6 Mon Sep 17 00:00:00 2001
From: Vadim Rutkovsky <vrutkovs@redhat.com>
Date: Wed, 25 Mar 2015 15:26:45 +0100
Subject: [PATCH] Fix builddir!=srcdir issue

---
 src/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index 8563031..cd8ae8d 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -23,7 +23,7 @@ tali_SOURCES =       \
 BUILT_SOURCES = tali-resources.c
 
 tali-resources.c: $(shell $(GLIB_COMPILE_RESOURCES) --generate-dependencies tali.gresource.xml)
-	$(AM_V_GEN) $(GLIB_COMPILE_RESOURCES) --target=$@ --sourcedir=$(srcdir) --generate-source tali.gresource.xml
+	$(AM_V_GEN) $(GLIB_COMPILE_RESOURCES) --target=$@ --sourcedir=$(builddir) --sourcedir=$(srcdir) --generate-source $(srcdir)/tali.gresource.xml
 
 tali_CPPFLAGS = \
 	-I$(top_srcdir) \
-- 
1.8.3.1

